### ISSUE TITLE
Using mara_page without mara_acl and without patching permissions leads to TypeError

### ISSUE THREAD

**Issue by mmanhertz** [mara/mara-page#7]
There seems to be an issue in `mara_page/acl.py`, where if you don't also use `mara_acl` or patch `current_user_has_permissions` you'll get a `TypeError: 'map' object is not subscriptable`:

```
...
[2020-10-14 12:16:20,247] ERROR in app: Exception on /pipelines/dependency-graph [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mara/app/src/mara-page/mara_page/acl.py", line 101, in wrapper
    if not current_user_has_permission(resource):
  File "/mara/app/src/mara-page/mara_page/acl.py", line 47, in current_user_has_permission
    return current_user_has_permissions([resource])[0][1]
TypeError: 'map' object is not subscriptable
172.19.0.1 - - [14/Oct/2020 12:16:20] "GET /pipelines/dependency-graph HTTP/1.1" 500 -
...
```

I "fixed" it locally by changing `current_user_has_permissions` to
```
def current_user_has_permissions(resources: [AclResource]) -> [[AclResource, bool]]:
    """
    Determines whether the currently logged in user has permissions for a list of resources.
    Implement actual behavior by patching the function.
    """
    return list(map(lambda resource: [resource, True], resources))
```

**Comment by jankatins** [comment]
This is in https://github.com/mara/mara-page/pull/6

**Comment by gathineou** [comment]
Hi! Above PR is merged (release 1.5.2) so I will close this issue. Thanks both!
